### PR TITLE
238 recognisable experiences 1

### PR DESCRIPTION
**Pull request by Nikomus**
Trello card: https://trello.com/c/62AzDRa2/238-recognisable-experiences-stage-i

## Description

- Fills in the markup for the grid of quote cards
- Some basic styles

@neontribe/sea-map

### CODE REVIEW

File: src/Page/NotAlone.elm [hunk 1]
```diff
@@ -29,6 +32,79 @@ update msg model =
...
+            , withMedia [ only screen [ Media.minWidth (px 576) ] ]
+                [ flex3 zero zero desktopWidth ]
+            ]
+        ]
+        [ span [ css [ quoteStyle ] ] [ text quote ]
+        , span [ css [ details ] ] [ text ("- " ++ name ++ ", " ++ age) ]
+        ]
+
+
+desktopWidth =
```
Comment (katjam) [inline src/Page/NotAlone.elm:73]: Should these be in `Theme.elm` ?

File: src/Page/NotAlone.elm [hunk 2]
```diff
@@ -29,6 +32,79 @@ update msg model =
...
+
+
+quoteStyle : Style
+quoteStyle =
+    batch
+        [ Css.flex3 zero zero (pct 60)
+        ]
+
+
+details : Style
```
Comment (katjam) [inline src/Page/NotAlone.elm]: `detailsStyle` for consistancy?

File: src/Page/NotAlone.elm [hunk 3]
```diff
@@ -29,6 +32,79 @@ update msg model =
...
+grid =
+    batch
+        [ displayFlex
+        , flexWrap wrap
+        , justifyContent spaceBetween
+        ]
+
+
+card : String -> String -> String -> Html msg
+card quote name age =
```
Comment (katjam) [inline src/Page/NotAlone.elm:60]: Will this be the whole card? Or `cardFront`?

**Review by katjam — APPROVED** [review]
Look great. I had a couple comments but nothing show-stopping.

File: src/Page/NotAlone.elm (hunk 1 above)
Comment (Nikomus) [reply src/Page/NotAlone.elm:73]: Maybe with better names. They're specific to the card widths here but we might want some similar styling with the resources grid. Abstract and rename with that card if it works out that way?

File: src/Page/NotAlone.elm (hunk 2 above)
Comment (Nikomus) [reply src/Page/NotAlone.elm]: This is a bit annoying. I had tried to not pad all the style functions out with -Style suffixes before but a lot of names are already in use.

Comment (Nikomus) [reply src/Page/NotAlone.elm]: I can't see a particular Elm convention for function naming based on return type - should we decide one? Would it be easier if all style functions ended `Style`?

File: src/Page/NotAlone.elm (hunk 3 above)
Comment (Nikomus) [reply src/Page/NotAlone.elm:60]: Whole card for now :)

File: src/Page/NotAlone.elm (hunk 1 above)
Comment (katjam) [reply src/Page/NotAlone.elm:73]: :+1:

File: src/Page/NotAlone.elm (hunk 3 above)
Comment (katjam) [reply src/Page/NotAlone.elm:60]: There will be 2 more quotes but we can abstract once they start taking Keys.

File: src/Page/NotAlone.elm (hunk 2 above)
Comment (katjam) [reply src/Page/NotAlone.elm]: Yeah - I was just going with the convention you set by the 2 above it.

Comment (Nikomus) [reply src/Page/NotAlone.elm]: There will be styles in Theme and Definition that don't, but I can make everything have it.

**Review by katjam — APPROVED** [review]
Looks great! :umbrella:

**Pull request merged**